databuilder/extractor/bigquery_usage_extractor.py

```diff
@@ -89,6 +93,12 @@ def _count_usage(self):
                 # https://cloud.google.com/logging/docs/reference/audit/bigquery/rest/Shared.Types/AuditData#JobStatistics
                 continue
 
+            # if email filter is provided, only the email matched with filter will be recorded.
+            if self.email_filter:
+                if not re.match(self.email_filter, email):
+                    # the usage account not match filter
```

Comment:
so it means user needs to provide an email pattern, and it will only process emails that follow the pattern? Should it be named as email_pattern or something else? The `EMAIL_FILTER` looks to me like filtering out those emails.

Reply:
sure. updated